src/ncp/ncp_base_mtd.cpp

```diff
@@ -3657,6 +3657,8 @@ void NcpBase::ProcessThreadChangedFlags(void)
         {OT_CHANGED_THREAD_PANID, SPINEL_PROP_MAC_15_4_PANID},
         {OT_CHANGED_THREAD_NETWORK_NAME, SPINEL_PROP_NET_NETWORK_NAME},
         {OT_CHANGED_THREAD_EXT_PANID, SPINEL_PROP_NET_XPANID},
+        {OT_CHANGED_THREAD_RLOC_ADDED, SPINEL_PROP_IPV6_ADDRESS_TABLE},
```

Comment:
update the comment to reflect this
https://github.com/openthread/openthread/blob/master/src/ncp/ncp_base_mtd.cpp#L3742

Reply:
Since we are touching this part, how about aggregating: OT_CHANGED_THREAD_RLOC_ADDED and OT_CHANGED_THREAD_RLOC_REMOVED into a single OT_CHANGED_THREAD_RLOC

Reply:
> update the comment to reflect this

Sure. Changed to `OT_CHANGED_SECURITY_POLICY`.

Reply:
> Since we are touching this part, how about aggregating: OT_CHANGED_THREAD_RLOC_ADDED and OT_CHANGED_THREAD_RLOC_REMOVED into a single OT_CHANGED_THREAD_RLOC

We also have `OT_CHANGED_IP6_ADDRESS_REMOVED` and `OT_CHANGED_IP6_ADDRESS_ADDED`. Strictly speaking, we should just use `OT_CHANGED_IP6_ADDRESS` because I don't see why `RLOC` is not an `IP6 ADDRESS`. 
The purpose of this PR is to just update RLOC on `wpantund`. Maybe we can enhance it in another PR. Thoughts?